retail-managment-system/rms-ui/src/app/core/layout/side-bar/side-bar.component.html

```diff
@@ -43,9 +38,4 @@
       >Products</a
     >
   </p>
-  <p>
-    <a mat-button routerLink="profits" routerLinkActive="active" class="fs-5"
-      >Profits</a
-    >
-  </p>
```

Comment:
why deleted profits?

and why deleted every thing related with profits?

Reply:
I did not commit changes related to it , but now I did